application/backup.sh

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env sh
+# Error Tracking
+if [ ! -z "${ SENTRY_DSN }"]; then
+    eval "$(curl -sL https://sentry.io/get-curl.sh)"
```

Comment:
From [this page](https://blog.sentry.io/sentry-bash/), I think it needs to be `get-cli`. But you aren't using the sentry CLI anyway, so you can delete this line if you don't want to use it.
```suggestion
    eval "$(curl -sL https://sentry.io/get-cli)"
```